components/splitter/overview.md

```diff
@@ -12,72 +12,55 @@ position: 0
 
 The <a href="https://www.telerik.com/blazor-ui/splitter" target="_blank">Blazor Splitter component</a> lets you divide a portion of the page into several pieces that the user can resize and collapse. This provides real estate management for the app and the end user so they can focus on the content that is important in their current task. You can also [save and load its state]({%slug splitter-state%}), and respond to [events]({%slug splitter-events%}).
 
-## Creating Splitter for Blazor
+## Creating Blazor Splitter
 
-1. Declare the `<TelerikSplitter>` tag
+1. Add the `<TelerikSplitter>` tag.
+1. Set the `Width` and `Height` parameters in any valid CSS unit. See [Splitter Parameters](#splitter-parameters) for more information about the component behavior when dimensions are not set.
+1. Inside the `<SplitterPanes>` child tag, add the desired `<SplitterPane>` tags to create sections of content.
+1. Inside each `<SplitterPane>`, add the desired content as HTML markup or other components.
+1. [Configure the panes, for example, set their `Size`, `Min`, `Max`, `Collapsible` and `Resizable` parameters]({%slug splitter-panes%}#pane-parameters). By default, Splitter panes are resizable, but not collapsible.
```

Comment:
If the entire sentence is a link, it's best to include the ending punctuation in the link as well.

```suggestion
1. [Configure the panes. For example, set their `Size`, `Min`, `Max`, `Collapsible` and `Resizable` parameters.]({%slug splitter-panes%}#pane-parameters) By default, Splitter panes are resizable, but not collapsible.
```